src/cli/apply_test.py

```diff
@@ -4,13 +4,11 @@
 from pathlib import Path
 from unittest.mock import patch
 
-import pytest
 from click.testing import CliRunner
 
 from src.cli.apply import Apply
 
 
-@pytest.mark.only
```

Comment:
Add a precommit hook to grep across the files to ensure that this isnt' committed.

Also add a precommit hook to search for "DO NOT COMMIT"